docs/usage.md

```diff
@@ -712,11 +712,11 @@ You can then add the `<YOUR_DB_NAME>/` path to your nf-core/taxprofiler database
 
 See the [MALT manual](https://software-ab.informatik.uni-tuebingen.de/download/malt/manual.pdf) for more information.
 
-#### MetaPhlAn3 custom database
+#### MetaPhlAn custom database
 
-MetaPhlAn3 does not allow (easy) construction of custom databases. Therefore we recommend to use the prebuilt database of marker genes that is provided by the developers.
+MetaPhlAn does not allow (easy) construction of custom databases. Therefore we recommend to use the prebuilt database of marker genes that is provided by the developers.
```

Comment:
We should also mention here that every version of MetaPhlAn essentially corresponds to a new version of the database, and (Also here) that if you download the MetaPhlAn3 database, you need to supply `--mpa3` to the database parameters.